yarn.lock

```diff
@@ -1583,7 +1583,7 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@dnnsoftware/dnn-react-common@9.9.1, @dnnsoftware/dnn-react-common@workspace:Dnn.AdminExperience/ClientSide/Dnn.React.Common":
+"@dnnsoftware/dnn-react-common@9.10.1, @dnnsoftware/dnn-react-common@workspace:Dnn.AdminExperience/ClientSide/Dnn.React.Common":
```

Comment:
Is this normal or what is going on ?

Reply:
We should probably run `yarn install` when we bump the version so that the lock file gets the current version. This change isn't necessarily related to updating Yarn itself, but every time an install is run it's going to update the version of the workspace in `yarn.lock` to match `package.json`.

Reply:
Oh, yeah I see, cool. It's not a big deal for this PR, but yeah, let's improve that as another PR